modules/radio/css/lib/radio.scss

```diff
@@ -3,13 +3,13 @@
 
   input + label::before {
     border-radius: $wdc-border-radius-circle;
-    background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg'%3E%3Cg%3E%3Ccircle fill='%23FFFFFF' cx='12' cy='12' r='4'%3E%3C/circle%3E%3C/g%3E%3C/svg%3E");
+    background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg'%3E%3Cg%3E%3Ccircle fill='%23FFFFFF' cx='9' cy='9' r='4.5'%3E%3C/circle%3E%3C/g%3E%3C/svg%3E");
```

Comment:
Thank you for your contribution! Is there any reasons why you increase the radio of the white dot from 4 to 4.5? It looks like 4 is matching the react component whereas 4.5 is a tiny bit bigger.

Reply:
I tested it and it is a bit bigger. We don't have a visual regression with the radio activated, so it wasn't caught. My guess is `4.5` was chosen because it is half of `9`?

Reply:
@stephanerangaya @NicholasBoll 
4.5 was chosen because it is half of 9.  When I tested in various browsers with it at 4, it still appeared to be slightly off.  Let me know if you feel the size should remain at 4 and the slight difference is marginal enough to resolve this issue.

Reply:
@erik-vanlankvelt Which browser(s) did you notice an issue with 4px? I think we would prefer to keep it at 4px since 4.5px would put make it subpixel.

Reply:
@anicholls 
I've updated the size to be 4px again.  I noticed a slight issue with the positioning of the white dot in Chrome and Safari at 75%.
![Chrome 75%](https://user-images.githubusercontent.com/873400/73128780-c4a24c80-3f91-11ea-85a0-271b7e84b888.png)
![Safari 75%](https://user-images.githubusercontent.com/873400/73128781-c53ae300-3f91-11ea-8f52-e6bad76f3596.png)

Reply:
@anicholls please let me know if you feel there is more to be done here, thanks!